packages/apm/src/span.ts

```diff
@@ -175,10 +175,11 @@ export class Span implements SpanInterface, SpanContext {
   }
 
   /**
-   * Creates a new `Span` while setting the current `Span.id` as `parentSpanId`.
-   * Also the `sampled` decision will be inherited.
+   * @inheritDoc
    */
-  public child(spanContext?: Pick<SpanContext, Exclude<keyof SpanContext, 'spanId'>>): Span {
+  public child(
+    spanContext?: Pick<SpanContext, Exclude<keyof SpanContext, 'spanId' | 'sampled' | 'traceid' | 'parentSpanId'>>,
```

Comment:
I suspect this is case-sensitive?

```suggestion
    spanContext?: Pick<SpanContext, Exclude<keyof SpanContext, 'spanId' | 'sampled' | 'traceId' | 'parentSpanId'>>,
```